crates/typst/src/text/raw.rs

```diff
@@ -231,12 +232,22 @@ pub struct RawElem {
     /// #let hi = "Hello World"
     /// ```
     /// ````
+    ///
+    /// Additionally, you can set the theme to `none` to disable highlighting.
+    ///
+    /// ````example
+    /// #set raw(theme: none)
+    ///
+    /// ```typ
+    /// #let this = "is not highlighted"
+    /// ```
+    /// ````
```

Comment:
While it'd be nice to have an example, unfortunately parameters can have only one example (which is by default hidden in a drop down). But I think people can imagine how it looks without highlighting, so I guess it's not critical to have this.